config/helm/delete-rbac-proxy.yaml

```diff
@@ -0,0 +1,73 @@
+# We are not deploying rbac-proxy (it has been removed in upstream as well).
```

Comment:
Patch to delete everything related to RBAC proxy. This is also removed from upstream in newer versions.